content/installation/setup/configuration/DistributedConfiguration.md

```diff
@@ -169,27 +169,28 @@ The following steps outline the process to migrate Contrast to your own Tomcat i
 
 In the example below, Contrast was installed at path `/usr/local/contrast`. Gather the following files: 
 
+* data/agents/
 * data/conf/
 * data/esapi/
 * data/.contrast
 * data/.initialized
 * data/cache/
 * data/contrast.lic
-* webapps/Contrast.war
+* webapp/Contrast.war
 
-This code compresses necessary artifacts into your user's home directory:
+The following commands compresses necessary artifacts into your user's home directory:
 
 ```
 $ cd /usr/local/contrast
-$ tar -czvf ~/ctdc.tar.gz data/conf data/contrast.lic data/esapi/ data/cache/ data/.initialized data/.contrast webapps/Contrast.war
+$ tar -czvf ~/ctdc.tar.gz data/agents data/conf data/contrast.lic data/esapi/ data/cache/ data/.initialized data/.contrast webapp/Contrast.war
 ```
 
 ### Install Tomcat and Java
 
 This process varies based on your operating system. You must install:
 
 * Tomcat 7 (Contrast recommends Tomcat 7.0.61)
-* Java 7 (Contrast recommends Java 1.7.0_80)
+* TeamServer supported Java as shown in our [System Requirements](installation-setup.html#contrast-reqs)
```

Comment:
Please change bulleted item to: 
"Supported versions of Java, as shown in [System Requirements](installation-setup.html#contrast-reqs)